samples/Samples.h

```diff
@@ -6,31 +6,31 @@ Shared include file for the samples
 
 #pragma once
 
-#ifdef  __cplusplus
+#ifdef __cplusplus
 extern "C" {
 #endif
 
 #include <com/amazonaws/kinesis/video/webrtcclient/Include.h>
 
-#define NUMBER_OF_H264_FRAME_FILES                                              403
-#define NUMBER_OF_OPUS_FRAME_FILES                                              618
-#define DEFAULT_FPS_VALUE                                                       25
-#define DEFAULT_MAX_CONCURRENT_STREAMING_SESSION                                10
+#define NUMBER_OF_H264_FRAME_FILES               403
+#define NUMBER_OF_OPUS_FRAME_FILES               618
+#define DEFAULT_FPS_VALUE                        25
+#define DEFAULT_MAX_CONCURRENT_STREAMING_SESSION 10
```

Comment:
`AlignConsecutiveAssignments: true` may prevent this formatting

Reply:
It will. But I am going to let it stay.